pkg/proxy/handlers/spacelister_test.go

```diff
@@ -33,7 +33,14 @@ func buildSpaceListerFakes(t *testing.T) (*fake.SignupService, *test.FakeClient)
 		newSignup("parentspace", "parent.space", true),
 		newSignup("childspace", "child.space", true),
 		newSignup("grandchildspace", "grandchild.space", true),
-	)
+	}
+	if publicViewerEnabled {
+		signups = append(signups,
+			newSignup("communityspace", "community.space", true),
+			newSignup("communitylover", "community.lover", true),
```

Comment:
just curious - what is this signup used for ? I don't see any SB for it. Is it just noise ?

Reply:
I tried to simplify this a bit in a0cc9fd14bbb9af12011649875b02b4898169cc6

Reply:
and in 2f018f851de0b6e0cf9bac9c456207b4e5932cf1